### DISCUSSION TITLE
http/1.1 internal server error 43531

### DISCUSSION THREAD

**Discussion by willperring** [area17/twill#1044] [Q&A]
Has anyone else encountered this?

Intermittently, I get a 'http/1.1 internal server error 43531', which is literally just that, black text on white. Nothing else. On refresh, the page is normal.

I've never seen it in any other PHP project, but I've not seen it in two different Twill applications, on different servers and environments where twill is the only common factor. That's not to say Twill is the issue, but it seems like the best place to ask.

I can't even find mention of it online, beyond the scope of NetScaler/Citrix.

Am I going mad? Has anyone else encountered this?

**Comment by ifox** [comment]
Hi @willperring, I'm afraid I've never seen this error before and I'd be surprised if Twill does anything producing this that any other Laravel app wouldn't, but we never know, are you doing anything custom in your instance?
